noir-projects/noir-contracts/contracts/fpc_contract/src/config.nr

```diff
@@ -0,0 +1,23 @@
+use dep::aztec::protocol_types::{address::AztecAddress, traits::{Deserialize, Serialize}};
+
+global CONFIG_LENGTH: u32 = 2;
+
+pub struct Config {
```

Comment:
This was originally called Settings but I renamed it to keep it consistent with naming in the AMM.